src/schema/v2/homeView/HomeViewSection.ts

```diff
@@ -164,6 +168,53 @@ const HomeViewArticlesSectionType = new GraphQLObjectType<any, ResolverContext>(
   }
 )
 
+export const ExploreByMarketingCategoriesType = new GraphQLObjectType<
```

Comment:
I can work on renaming the ones I've added but I won't touch the existing ones since it creates breaking changes(not breaking the schema per see but the element it's going to disappear on Eigen until we wrap up the new type in the sections) in the app versions using the older schemas(case of `HomeViewSectionMarketingCollections`). We can file an extra ticket to unify the `HomeViewSectionMarketingCollections` if you want.

change some names here based on your suggestion https://github.com/artsy/metaphysics/pull/6058/commits/3fead7c8b81da5733a8ae62a7e6bea8bc7d46cdc

Reply:
💯